src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/MetadataBuilder.Tables.cs

```diff
@@ -302,15 +306,18 @@ public void SetCapacity(TableIndex table, int capacity)
             });
         }
 
-        public void AddInterfaceImplementation(
+        public InterfaceImplementationHandle AddInterfaceImplementation(
             TypeDefinitionHandle type,
             EntityHandle implementedInterface)
         {
             _interfaceImplTable.Add(new InterfaceImplRow
             {
                 Class = (uint)MetadataTokens.GetRowNumber(type),
-                Interface = (uint)CodedIndex.ToTypeDefOrRef(implementedInterface)
+                Interface = (uint)CodedIndex.ToTypeDefOrRefOrSpec(implementedInterface)
             });
+
+            // TODO:
+            return (InterfaceImplementationHandle)MetadataTokens.Handle(TableIndex.InterfaceImpl, _interfaceImplTable.Count);
```

Comment:
This TODO can be fixed now, right? MetadataTokens.InterfaceImplementationHandle is in IIRC.